deployments/deploy-ssh.sh

```diff
@@ -763,6 +763,12 @@ spec:
       value: "${PX_BACKUP_MONGODB_PASSWORD}"
     - name: ENABLE_GRAFANA
       value: "${ENABLE_GRAFANA}"
+    - name: EKS_CLUSTER_NAME
```

Comment:
can you try if these values can get populated automatically

Reply:
Sure.